src/metabase/public_settings.clj

```diff
@@ -175,7 +175,8 @@
 (defsetting enable-nested-queries
   (deferred-tru "Allow using a saved question as the source for other queries?")
   :type    :boolean
-  :default true)
+  :default true
+  :visibility :authenticated)
```

Comment:
The only backend change here is changing the `enable-nested-queries` setting's visibility from `:admin` to `:authenticated`. This is needed so the FE can take in into account when a non-admin user is signed in